README.md

```diff
@@ -1,2 +1,48 @@
-# vet_clinic1
-creating a database named Animals.  
+# curriculum-databases-projects-template
```

Comment:
### Optional
- [ ] It would be better if we customize our README file so that it reflect the current project 💯 Also, it would be better if we add instructions about how to clone and run the app locally in the README file so that non technical people can appreciate the project